crates/sui-core/src/authority/authority_store.rs

```diff
@@ -298,11 +300,11 @@ impl<S: Eq + Serialize + for<'de> Deserialize<'de>> SuiDataStore<S> {
     /// the same certificate may be written twice (but that is OK since it is valid.)
     pub fn add_pending_certificates(
         &self,
-        certs: Vec<(TransactionDigest, CertifiedTransaction)>,
+        certs: Vec<(TransactionDigest, Option<CertifiedTransaction>)>,
```

Comment:
while you're at it, maybe change this to take a `&[]` instead of Vec, since you're creating a vec just to call this in one place?